### ISSUE TITLE
Too many pixel values emitted for sliced 3D images

### ISSUE THREAD

**Issue by rsrock** [timholy/ImageView.jl#65]
Continuing the thread in #53, here's a minimal test case

```
using Images, ImageView
foo = rand(10, 10, 10)
view(sliceim(Image(foo), :,:,8))
```

And then mouse over the image.  I get:

![untitled](https://cloud.githubusercontent.com/assets/1935770/6629859/591276e8-c8e0-11e4-9f09-232141f7f6be.png)

Notice that I get 10 pixel values.  The image suddenly gets stretched too.

**Comment by timholy** [comment]
Yep, I see that too. I'll try to investigate, but I'm traveling now and for the next week, so no promises.

**Comment by rsrock** [comment]
No worries, I have a good workaround.  Just making note of it (and a few other things you may have seen).  This one does look like a funny one, though.

**Comment by rsrock** [comment]
Took a quick peek at this again in some spare time.  Here's a band-aid for this issue.  Replace line 532 `val = img2[xi, yi]` with:

```
val = img2.imslice[xi, yi]
```

But that doesn't seem entirely correct, it seems like `ImageSlice2d` should have a direct way of handling sliced images in there.

**Comment by timholy** [comment]
Closed by #107
